### PR TITLE
[ADD] New module to add tags on products

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Hi  @KolushovAlexandr, Thanks for your contribution.
I think you reused this V10.0 module ? https://github.com/OCA/product-attribute/pull/274/files

could you remake your PR, using this procedure ? 
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

so,
- it will be more easy to review only the changes you did
- it respects original authors in the commits.

Do not hesitate to ask if it's not clear.

kind regards.

**Comment by yelizariev** [comment]
@legalsylvain he actually used that procedure.

**Comment by legalsylvain** [comment]
@yelizariev : well, there was only one commit before my first comment so, no completely ;-)

**Comment by yelizariev** [comment]
Why runbot is red?

**Comment by KolushovAlexandr** [comment]
Could anyone review this PR? Which further steps does it require?

**Review by legalsylvain — APPROVED** [review]
:+1: 

- Code review (actually, no changes;-) ) 
- No test.

**Comment by KolushovAlexandr** [comment]
@legalsylvain  All checks are green, tests were added, may it be merged?

**Comment by legalsylvain** [comment]
Hi @KolushovAlexandr. 

thanks for you contribution. About the merge in the official repositories, here is the rule of OCA : 

*"Your code will be merged upon 3 positive reviews within 5 days (or 2 after more than 5 days). At least one of the review above must be from a member of the PSC or someone from the OCA Core Maintainer."*

more information here : https://odoo-community.org/page/code

kind regards.

**Comment by yelizariev** [comment]
@rafaelbn, @benwillig, @leemannd, @grindtildeath, @lreficent  (guys from https://github.com/OCA/product-attribute/pull/274)
Could you please review this PR? It was made almost year ago

**Review by benwillig — APPROVED** [review]
LGTM. Thanks for the migration

File: product_template_tags/tests/test_product_template_tags.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestProductTemplateTag(TransactionCase):
+    pre_install = True
+    post_install = True
```
Comment (grindtildeath) [inline product_template_tags/tests/test_product_template_tags.py:10]: Any reason for these two flags ?

File: product_template_tags/tests/test_product_template_tags.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from odoo.tests.common import TransactionCase
```
Comment (grindtildeath) [inline product_template_tags/tests/test_product_template_tags.py:5]: Adding an header with license would be nice :wink:

**Review by grindtildeath — APPROVED** [review]
To make it complete, you could also split the README into parts, and add yourself as a contributor, but overall looks good :+1:

File: product_template_tags/tests/test_product_template_tags.py (hunk 1 above)
Comment (yelizariev) [reply product_template_tags/tests/test_product_template_tags.py:10]: By default, it tests in following mode:

```
at_install = True
post_install = False
```
So, those flags explicitly test in all modes

**Comment by yelizariev** [comment]
@KolushovAlexandr why did you close the PR?

**Pull request closed without merging**